sdk/attestation/azure-security-attestation/cgmanifest.json

```diff
@@ -1,36 +1,47 @@
 {
-    "Registrations": [
-        {
-            "Component": {
-                "Type": "git",
-                "git": {
-                    "RepositoryUrl": "https://github.com/google/googletest",
-                    "CommitHash": "703bd9caab50b139428cea1aaff9974ebee5742e"
-                }
-            },
-            "DevelopmentDependency": true
-        },
-        {
-            "Component": {
-                "Type": "other",
-                "Other": {
-                    "Name": "clang-format",
-                    "Version": "9.0.0-2",
-                    "DownloadUrl": "https://ubuntu.pkgs.org/18.04/ubuntu-updates-universe-amd64/clang-format-9_9-2~ubuntu18.04.2_amd64.deb.html"
-                }
-            },
-            "DevelopmentDependency": true
-        },
-        {
-            "Component": {
-                "Type": "other",
-              "Other": {
-                "Name": "doxygen",
-                "Version": "1.8.20",
-                "DownloadUrl": "https://azuresdktooling.blob.core.windows.net/dependencies/doxygen-1.8.18-setup.exe"
-              }
-            },
-            "DevelopmentDependency": true
+  "Registrations": [
+    {
+      "Component": {
+        "Type": "other",
+        "Other": {
+          "Name": "openssl",
+          "Version": "1.1.1n",
+          "DownloadUrl": "https://www.openssl.org/source/openssl-1.1.1n.tar.gz"
```

Comment:
Could this be done as a git registration instead of "other"? If it helps, the commit tag for OpenSSL 1.1.1n is `d82e959e621a3d597f1e0d50ff8c2d8b96915fd7`.

Reply:
done